WalletWasabi/Services/SystemAwakeChecker.cs

```diff
@@ -66,51 +66,54 @@ private SystemAwakeChecker(CoinJoinManager coinJoinManager, Func<Task<IPowerSavi
 
 		protected override async Task ActionAsync(CancellationToken cancel)
 		{
-			IPowerSavingInhibitorTask? task = _powerSavingTask;
-
 			switch (CoinJoinManager.HighestCoinJoinClientState)
 			{
 				case CoinJoinClientState.Idle:
-					if (task is not null)
-					{
-						Logger.LogWarning("Computer idle state is allowed again.");
```

Comment:
Why not change it to `Logger.LogTrace(...)`?

Reply:
i forgot about `LogTrace`, will change it!